redash/query_runner/cass.py

```diff
@@ -77,10 +94,14 @@ def run_query(self, query, user):
             if self.configuration.get('username', '') and self.configuration.get('password', ''):
                 auth_provider = PlainTextAuthProvider(username='{}'.format(self.configuration.get('username', '')),
                                                       password='{}'.format(self.configuration.get('password', '')))
-                connection = Cluster([self.configuration.get('host', '')], auth_provider=auth_provider, protocol_version=3)
+                connection = Cluster([self.configuration.get('host', '')],
+                                     auth_provider=auth_provider,
+                                     protocol_version=self.configuration.get('protocol', ''),
+                                     cql_version=self.configuration.get('cqlversion', ''))
```

Comment:
Are you sure this one is needed? According to the docs it knows to pick the right one: "If a specific version of CQL should be used, this may be set to that string version. Otherwise, the highest CQL version supported by the server will be automatically used." (https://datastax.github.io/python-driver/api/cassandra/cluster.html)

If it is, the default value needs to be `None` and not an empty string.